src/Dapr.AspNetCore/DaprEndpointRouteBuilderExtensions.cs

```diff
@@ -21,6 +25,39 @@ namespace Microsoft.AspNetCore.Builder
     /// </summary>
     public static class DaprEndpointRouteBuilderExtensions
     {
+        /// <summary>
+        /// Adds Dapr integration for Minimal APIs to the provided <see cref="IEndpointRouteBuilder" />.
+        /// </summary>
+        /// <param name="builder">The <see cref="IEndpointRouteBuilder" />.</param>
+        /// <param name="services">The <see cref="IServiceCollection" />.</param>
+        /// <param name="configureClient">The (optional) <see cref="DaprClientBuilder" /> to use for configuring the DaprClient.</param>
+        /// <returns>The <see cref="IEndpointRouteBuilder" /> builder.</returns>
+        public static IEndpointRouteBuilder AddDapr(this IEndpointRouteBuilder builder, IServiceCollection services, Action<DaprClientBuilder> configureClient = null)
```

Comment:
The design of this does not really make sense to me. I would expect the **services** to already be baked by the time you have an endpoint route builder. 

It's still not super clear to me what (if any) changes are needed and what scenarios are being enabled. Let's use the issue to discuess.